react-components/src/utilities/queryKeys.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type IdEither } from '@cognite/sdk/';
+
+export const queryKeys = {
+  all: ['cdf'] as const,
+  // ASSETS
+  assetsById: (ids: IdEither[]) => [...assets(), ids] as const,
+  // TIMESERIES
+  timeseriesById: (ids: IdEither[]) => [...timeseries(), ids] as const,
+  timeseriesLatestDatapoint: () => [...timeseries(), 'latest-datapoints'] as const,
+  // TIMESERIES RELATIONSHIPS WITH ASSETS
+  timeseriesLinkedToAssets: () =>
+    [...queryKeys.all, 'timeseries', 'timeseries-linked-assets'] as const
+} as const;
+
+const assets = (): string[] => {
+  return [...queryKeys.all, 'assets'];
```

Comment:
Nitpick - this (and the `timeseries` lambda below) could just be constants instead of functions

Reply:
ok but I thought that we were using this as a pattern. I can change it

Reply:
https://github.com/cognitedata/reveal/pull/4414/commits/88550354c2c0f4e426f590f68eb38e3959679926